lib/msal-angular/src/msal-guard.service.ts

```diff
@@ -27,6 +27,28 @@ export class MsalGuard implements CanActivate {
         private broadcastService: BroadcastService
     ) {}
 
+    /**
+     * Builds the absolute url for the destination page
+     * @param path Relative path of requested page
+     * @returns Full destination url
+     */
+    getDestinationUrl(path: string): string {
+        // Absolute base url for the application
+        const baseUrl = this.location.normalize(document.getElementsByTagName("base")[0].href);
```

Comment:
Is this element always available?

Reply:
The Angular CLI adds it be default, but yeah, there isn't a guarantee. I'll add a default if it isn't there.

Reply:
So Angular apps must either provide this base element, or provide the base path via the `APP_BASE_HREF` injection token. I'm not sure how to account for the `APP_BASE_HREF`, given if the Guard expects it to be injected, Angular will throw an error if it is not provided. So for now, I'll default to `window.location.origin`, and incrementally handle `APP_BASE_HREF`.